models/rgcn_semantic_encoder.py

```diff
@@ -19,10 +19,12 @@ def __init__(self, args):
 
         self.n_graph_attn_composition_layers = args.n_graph_attn_composition_layers
         self.output_size = self.transformer.config.hidden_size
+        
+        self.graph_output_size = args.graph_dim
```

Comment:
For consistency, can you rename this field to `graph_dim` too, and update `forward`? And also remove the new line above it.

Reply:
Thanks for your review !  Have changed and committed.